corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -233,6 +233,7 @@ hqDefine("cloudcare/js/form_entry/form_ui", function () {
         delete json.tree;
         Container.call(self, json);
         self.blockSubmit = ko.observable(false);
+        self.hasSubmitAttempted = ko.observable(false);
```

Comment:
This could just be a js boolean, not an observable, since there's no UI responding to it.

Reply:
I did try that at first but noticed that the `self.erroredQuestions()` stops getting triggered when clicking 'Submit' when I keep it just as a boolean and not an observable. Any thooughts on why that might be happening ?

Reply:
Sorry, you're totally right - it does need to be an observable, because it's accessed inside of a computed observable.